reflex/constants/event.py

```diff
@@ -11,6 +11,7 @@ class Endpoint(Enum):
     EVENT = "_event"
     UPLOAD = "_upload"
     AUTH_CODESPACE = "auth-codespace"
+    HEALTH = "health"
```

Comment:
```suggestion
    HEALTH = "_health"
```

recommend a leading underscore for this endpoint